src/org/opendatakit/briefcase/ui/reused/AggregateServerConnectionConfiguration.java

```diff
@@ -0,0 +1,68 @@
+package org.opendatakit.briefcase.ui.reused;
+
+import org.opendatakit.briefcase.reused.BriefcaseException;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.Objects;
+
+public class AggregateServerConnectionConfiguration {
+  private URL url;
+  private String username;
+  private String password;
+
+  public AggregateServerConnectionConfiguration(String url, String username, String password) {
```

Comment:
The constructor should receive a `URL` instance. If you need to make transformations to create an object, we encode that as a static factory method